recipe/conda_build_config.yaml

```diff
@@ -1,4 +0,0 @@
-c_compiler_version:    # [ppc64le]
-- '7'                  # [ppc64le]
-cxx_compiler_version:  # [ppc64le]
-- '7'                  # [ppc64le]
```

Comment:
Guessing since this doesn't fill out [all of these `zip_keys`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/46235e9609671b875b23bf8616d0fcbe4e180783/recipe/conda_build_config.yaml#L152-L160 ) re-rendering runs into issues. So trying dropping to see if re-rendering works (and maybe the build as well 🍀). If not, could also try filling out the missing keys here and re-rendering then

Reply:
Interesting still seeing the same re-rendering issue with this change